src/app/contribute/guidelines/component-guidelines/index.html

```diff
@@ -0,0 +1,269 @@
+  </p>
+
+  <p>
+    When your tests run, code coverage results are generated and can be located in <stache-code>coverage/&lt;browser version>/index.html</stache-code>. You can launch this straight from disk and view the SKY UX unit test code coverage results in your default web browser.
+  </p>
+
+  <stache-page-anchor>
+    Write visual regression tests
+  </stache-page-anchor>
+
+  <p>
+    During continuous integration builds, SKY UX runs visual regression tests through BrowserStack using <a stacheRouterLink="http://www.protractortest.org/">Protractor</a> and <a stacheRouterLink="https://github.com/koola/pix-diff">pix-diff</a>. These prerequisites are automatically installed when you run <stache-code>npm run test:visual</stache-code> to create and compare screenshots in the <stache-code>skyux-spa-visual-tests/screenshots-baseline-local/</stache-code> folder.
+  </p>
+
+  <p>
+    To create visual tests for a new component, create a folder for the tests in <stache-code>skyux-spa-visual-tests/src/app</stache-code> and add four files for the visual regression test:
+  </p>
+
+  <ul>
+    <li>
+      <stache-code>{{ '{' }}componentName}-visual.component.html</stache-code>: The template for the component to render and take a screenshot of.
+    </li>
+    <li>
+      <stache-code>{{ '{' }}componentName}-visual.component.ts</stache-code>: The TypeScript code for the component to render and take a screenshot of.
+    </li>
+    <li>
+      <stache-code>{{ '{' }}componentName}.visual-spec.ts</stache-code>: The Protractor code to run the screenshot tests.
+    </li>
+    <li>
+      <stache-code>index.html</stache-code>: The route file that contains the component to render and take a screenshot of.
+    </li>
+  </ul>
+
+  <p>
+    To run visual regression tests locally, you must first run them locally against the master branch and then run the tests against your feature branch. Keep in mind that you need to use the same monitor and resolution on both set of tests. If your visual regression tests fail, you can check the failed tests at <stache-code>skyux2\skyux-spa-visual-tests\screenshots-diff-local\diff</stache-code> to find the cause of the failures.
+  </p>
+
+  <stache-page-anchor>
+    Submit code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      Commit and push your changes to the repo.
+    </li>
+    <li>
+      Submit a pull request.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Available NPM scripts
+  </stache-page-anchor>
+
+  <p>
+    <stache-code>npm install</stache-code> and <stache-code>npm start</stache-code> are special commands. All other commands listed below should use the <stache-code>npm run COMMAND</stache-code> format. For example, <stache-code>npm run build</stache-code>.
+  </p>
+
+  <table class="content-table">
```

Comment:
These should be updated to only contain those listed at https://github.com/blackbaud/skyux-indicators/blob/master/package.json#L7-L12

Reply:
I imagine the Description should be column updated as well or was removing the extras fine? Either way, updated this.